distros/kubernetes/nvsentinel/templates/daemonset.yaml

```diff
@@ -41,8 +41,42 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       serviceAccountName: {{ include "nvsentinel.serviceAccountName" . }}
+      {{- if and .Values.global.datastore (eq .Values.global.datastore.provider "postgresql") }}
+      initContainers:
+        - name: fix-cert-permissions
+          image: "docker.io/bitnamilegacy/os-shell:12-debian-12-r30"
```

Comment:
Same comment as earlier about using the image name from values

Reply:
good call. will do.